applications/CoSimulationApplication/python_scripts/solver_wrappers/sdof/sdof_static_solver.py

```diff
@@ -0,0 +1,144 @@
+from __future__ import print_function, absolute_import, division  # makes these scripts backward compatible with python 2.6 and 2.7
+# Importing the base class
+import KratosMultiphysics
+from KratosMultiphysics.CoSimulationApplication.function_callback_utility import GenericCallFunction
+
+# Other imports
+import numpy as np
+import json
+import os
+
+class SDoFStaticSolver(object):
+    def __init__(self, input_name):
+
+        # mimicing two constructors
+        if isinstance(input_name, dict):
+            parameters = input_name
+
+        elif isinstance(input_name, str):
+            if not input_name.endswith(".json"):
+                input_name += ".json"
+
+            with open(input_name,'r') as ProjectParameters:
+                parameters = json.load(ProjectParameters)
+
+        else:
+            raise Exception("The input has to be provided as a dict or a string")
+
+        default_settings = {
+                "system_parameters":{
+                    "stiffness" : 4000.0
+                },
+                "initial_values":{
+                    "displacement"  : 0.0,
+                },
+                "boundary_conditions":{
+                    "external_load" : 5000.0
+                },
+                "solver_parameters": {
+                    "buffer_size"   : 1
+                },
+                "output_parameters":{
+                    "write_output_file": False,
+                    "file_name" : "sdof_static_solver/results_sdof.dat"
+                }}
+
+        RecursivelyValidateAndAssignDefaults(default_settings, parameters)
+
+        self.stiffness = parameters["system_parameters"]["stiffness"]
+
+        self.initial_displacement = parameters["initial_values"]["displacement"]
+
+        self.force = parameters["boundary_conditions"]["external_load"]
+
+        self.buffer_size = parameters["solver_parameters"]["buffer_size"]
+
+        self.output_file_name = parameters["output_parameters"]["file_name"]
+
+        self.write_output_file = parameters["output_parameters"]["write_output_file"]
+
+    def Initialize(self):
+        initial_values = self.initial_displacement
+        self.dx = initial_values
+
+        if self.write_output_file:
+            if os.path.isfile(self.output_file_name):
+                os.remove(self.output_file_name)
+            self.InitializeOutput()
+        self.time = 0.0
+
+    def InitializeOutput(self):
+        with open(self.output_file_name, "w") as results_sdof_static:
+            results_sdof_static.write("displacement" + "\n")
+        self.OutputSolutionStep()
+
+    def OutputSolutionStep(self):
+        if self.write_output_file:
+            with open(self.output_file_name, "a") as results_sdof_static:
+                #outputs results
+                results_sdof_static.write(str(self.dx) + "\n")
+
+    def AdvanceInTime(self, current_time):
+        self.time = 0.0
+        return self.time
```

Comment:
```suggestion
        return None
```

this will be needed in the future